autoflagging.user.js

```diff
@@ -272,38 +225,39 @@
    * It will use the results to decorate the Smokey reports which are already on the page.
    */
   autoflagging.callAPI = function (urls, page) {
-    //console.log("Call API");
+    // console.log("Call API");
     if (page == null) {
       page = 1;
     }
     var autoflagData = {};
     var url = autoflagging.baseURL + "&page=" + page + "&urls=" + urls;
-    //console.log("URL: " + url);
-    $.get(url, function(data) {
+    // console.log("URL: " + url);
+    $.get(url, function (data) {
       // Group information by link
       for (var i = 0; i < data.items.length; i++) {
         autoflagData[data.items[i].link] = data.items[i];
       }
 
       // Loop over all Smokey reports and decorate them
-      $(autoflagging.selector).each(function() {
+      $(autoflagging.selector).each(function () {
         var postURL = autoflagging.getPostURL(this);
         // TODO: show flag weight - first, the API needs to be changed
-        if (typeof autoflagData[postURL] == 'undefined')
+        if (typeof autoflagData[postURL] == "undefined") {
           return;
+        }
         autoflagging.decorateMessage($(this), autoflagData[postURL]);
         // Post deleted?
         if (autoflagData[postURL].deleted_at != null) {
-          $(this).find('.content').toggleClass('ai-deleted');
+          $(this).find(".content").toggleClass("ai-deleted");
         }
       });
 
       if (data.has_more) {
         // There are more items on the next 'page'
         autoflagging.callAPI(urls, ++page);
       }
-    }).fail(function(xhr, textStatus, error) {
-      autoflagging.notify('Failed to load data: ' + xhr.statusText);
+    }).fail(function (xhr, /* textStatus, error */) {
```

Comment:
Are you sure this is valid javascript? In google chrome I get `Uncaught SyntaxError: Unexpected token )` for `var test = function(a, /* b, c */) {};`